ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master.yaml

```diff
@@ -74,15 +74,7 @@ tests:
   commands: /tmp/run.sh
   container:
     from: logging-test-unit
-- as: full-integ-aws
-  commands: |
-    . openshift/ci-operator/build-image/setup-logging-for-e2e.sh
-  openshift_installer_src:
-    cluster_profile: aws
-- as: upgrade-full-integ-aws
-  commands: |
-    OPENSHIFT_BUILD_NAMESPACE= IMAGE_FORMAT= USE_CUSTOM_IMAGES=true hack/deploy-logging.sh
-    export LOGGING_DEPLOY_MODE=upgrade
-    . openshift/ci-operator/build-image/setup-logging-for-e2e.sh
+- as: e2e
```

Comment:
this is our `smoke test` currently right?
shouldn't we name it as such, or is the idea we'll eventually be building that up more in the future?

Reply:
I would expect we might want to add more tests, but I'm open to changing it.  the kickoff script I called e2e and thus the reason i chose this name

Reply:
i think its fine... though it does give us an opportunity to have a generic `smoke` entry point and then an `e2e` separate so we can determine more easily if its an issue with the image build/components standing up or how the components are being stressed

Reply:
updated to 'smoke' per slack convo